--- AttributeParser-org.pm	2010-05-08 17:08:16.000000000 +0200
+++ AttributeParser.pm	2010-05-08 17:11:16.000000000 +0200
@@ -11,11 +11,11 @@
 use vars qw ( @ISA );
 use strict;
 
-@ISA= qw ( Parse::Yapp::Driver );
+@ISA= qw ( Parse::Yapp::Driver::AttributeParser );
 #Included Parse/Yapp/Driver.pm file----------------------------------------
 {
 #
-# Module Parse::Yapp::Driver
+# Module Parse::Yapp::Driver::AttributeParser
 #
 # This module is part of the Parse::Yapp package available on your
 # nearest CPAN
@@ -29,7 +29,7 @@
 # (see the pod text in Parse::Yapp module for use and distribution rights)
 #
 
-package Parse::Yapp::Driver;
+package Parse::Yapp::Driver::AttributeParser;
 
 require 5.004;
 
